src/app/features/charts/components/score-barchart/score-barchart.component.html

```diff
@@ -1,7 +1,7 @@
-<div id="score-barchart">
+<div id="score-barchart" #div (window:resize)="onResize(div.getBoundingClientRect().width)">
```

Comment:
Get the width of the div when the window is resized, so the chart can also be resized

Reply:
As a suggestion, perhaps we can rename #div to #scoreBarChart.
We can also move the width logic out of the html and have the onResize method calculate the div width. 
 Then we have just presentation logic in the html.
Then we can do something like this in the html:
`<div id="score-barchart" #scoreBarChart>`

And in the component:
```
@HostListener('window:resize', ['$event.target'])
onResize() {
   const divWidth = this.scoreBarChart.getBoundingClientRect().width;
   ...
}
```

Reply:
Sure thing, done in the next commit